### ISSUE TITLE
[Request] Support Spark 3.5

### ISSUE THREAD

**Issue by melin** [microsoft/sql-spark-connector#249]
Hello,

Is it plan for a support of new release of spark 3.5 ?

Thanks

**Comment by atul-delphix** [comment]
Yes, please update here, waiting from long time and all the current version has lots of security vulnerabilities.

**Comment by miguruiz** [comment]
PR has been created https://github.com/microsoft/sql-spark-connector/pull/264/files
